pkg/datapath/linux/routing/migrate_test.go

```diff
@@ -0,0 +1,575 @@
+// Copyright 2016-2020 Authors of Cilium
```

Comment:
nit: Copyright year needs to be updated to 2021 I think. Same with other files touched in this PR.